src/Symfony/Bridge/Twig/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ CHANGELOG
 ---
 
  * Wrap help messages on form elements in `div` instead of `p`
+ * Implement the new `BlankableExtractorInterface` in the `TwigExtractor`, which allows extracting blank strings instead of a prefixed copy of the translation key
```

Comment:
This interface doesn't exist, right?

Reply:
Oops, good catch! Must have messed that up in the rebase. Fixed.